eng/helixcorefxtests.proj

```diff
@@ -0,0 +1,285 @@
+<Project InitialTargets="BuildHelixWorkItems" DefaultTargets="RunInParallelForEachScenario">
```

Comment:
Should this file be added manually?

Reply:
@marek-safar it looks like it was added manually in coreclr: https://github.com/dotnet/coreclr/commits/master/eng/helixcorefxtests.proj

the script correctly downloads the tests-manifest and parses it (`ParseBuildManifest` target) but then - I have no idea what it wants from me in order to download them. It was just a quick attempt